package.json

```diff
@@ -316,7 +316,8 @@
 	"resolutions": {
 		"@automattic/newspack-blocks/swiper@4.5.1": "patch:swiper@4.5.1#.patches/swiper@4.5.1.diff",
 		"@automattic/newspack-blocks/prettier": "npm:wp-prettier@2.2.1-beta-1",
-		"@types/react": "^17.0.33"
+		"@types/react": "^17.0.33",
+		"keytar@npm:7.7.0/node-addon-api": "3.1.0"
```

Comment:
`keytar` fails to compile when using `node-addon-api@3.2.1`.

Before #57633 this wasn't a problem since `keytar` depends on `node-addon-api@^3.0.0` and it was resolved as `3.1.0`. But after #57633 we have `node-addon-api@^3.0.0` and `node-addon-api@npm:^3.2.0`, which are both resolved as `3.2.1`. 

Using resolutions should fix the problem by forcing `keytar` to use `3.1.0`